app/routes/contracted-hours.js

```diff
@@ -42,16 +39,14 @@ module.exports = function (router) {
           woId: id,
           hoursUpdatedSuccess: req.query.hoursUpdatedSuccess,
           workloadType,
-          onOffenderManager: workloadType === PROBATION,
-          onCourtReports: workloadType === COURT_REPORTS
-
+          onOffenderManager: true
```

Comment:
good progress, it would be nice to see this go too in the future as presumably it drives a conditional statement somewhere